modin/core/io/file_dispatcher.py

```diff
@@ -338,3 +342,7 @@ def build_partition(cls, partition_ids, row_lengths, column_widths):
                 for i in range(len(partition_ids))
             ]
         )
+
+    @classmethod
+    def _file_not_found_msg(cls, filename: str):  # noqa: GL08
+        return f"No file with name '{filename}'"
```

Comment:
```suggestion
        return f"No such file: '{filename}'"
```
This way it looks more like the file not found error in python. What do people think?